<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {background: rgb(58, 94, 171);}

 QTabWidget::pane { /* The tab widget frame */
     border: 2px solid black;
 }

 /* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
 QTabBar::tab {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
     border: 2px solid #C4C4C3;
     border-bottom-color: #C2C7CB; /* same as the pane color */
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     min-width: 8ex;
     padding: 5px;
 }

 QTabBar::tab:selected, QTabBar::tab:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
 }

 QTabBar::tab:selected {
     border-color: #9B9B9B;
     border-bottom-color: #C2C7CB; /* same as pane color */
 }

 
caTable {
       font: 10pt;
       background: cornsilk;
       alternate-background-color: wheat;
}

caLineEdit {
     border-radius: 1px;
     background: lightyellow;
     color: black;
 }

caTextEntry {
    color: rgb(127, 0, 63);
    background-color: cornsilk;
    selection-color: #0a214c;
    selection-background-color: wheat;
    border: 1px groove black;
    border-radius: 1px;
    padding: 1px;
}

caTextEntry:focus {
    padding: 0px;
    border: 2px groove darkred;
    border-radius: 1px;
}

caSlider::groove:horizontal {
border: 1px solid #bbb;
background: lightgrey;
height: 20px;
border-radius: 4px;
}

caSlider::handle:horizontal {
background: red;
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 2px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="caThermo" name="cathermo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>61</width>
      <height>263</height>
     </rect>
    </property>
    <property name="channel" stdset="0">
     <string notr="true">ACM:TEST:AO</string>
    </property>
    <property name="look">
     <enum>caThermo::Limits</enum>
    </property>
    <property name="colorMode">
     <enum>caThermo::Alarm</enum>
    </property>
    <property name="limitsMode">
     <enum>caThermo::Channel</enum>
    </property>
    <property name="scaleValueEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="caNumeric" name="canumeric">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="channel" stdset="0">
     <string notr="true">ACM:TEST:AO</string>
    </property>
   </widget>
   <widget class="caTextEntry" name="catextentry_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="channel" stdset="0">
     <string>ACM:TEST:AO</string>
    </property>
   </widget>
   <widget class="caLineEdit" name="testvalue1">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="caLineEdit" name="calineedit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="channel" stdset="0">
     <string notr="true">ACM:TEST:AO</string>
    </property>
   </widget>
   <widget class="caSpinbox" name="caspinbox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="channel" stdset="0">
     <string notr="true">ACM:TEST:AO</string>
    </property>
   </widget>
   <widget class="caMeter" name="cameter">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>122</width>
      <height>122</height>
     </rect>
    </property>
    <property name="channel" stdset="0">
     <string>ACM:TEST:AO</string>
    </property>
    <property name="maxValue">
     <double>900.000000000000000</double>
    </property>
    <property name="limitsMode">
     <enum>caMeter::User</enum>
    </property>
    <property name="precisionMode">
     <enum>caMeter::User</enum>
    </property>
   </widget>
   <widget class="caThermo" name="testvalue6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>280</y>
      <width>61</width>
      <height>263</height>
     </rect>
    </property>
    <property name="channel" stdset="0">
     <string notr="true"/>
    </property>
    <property name="look">
     <enum>caThermo::Limits</enum>
    </property>
    <property name="colorMode">
     <enum>caThermo::Alarm</enum>
    </property>
    <property name="limitsMode">
     <enum>caThermo::Channel</enum>
    </property>
    <property name="scaleValueEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="caMeter" name="testvalue2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>122</width>
      <height>122</height>
     </rect>
    </property>
    <property name="channel" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="caNumeric" name="testvalue3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="channel" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="caSpinbox" name="testvalue4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="channel" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="caTextEntry" name="testvalue5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>360</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="channel" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="caLabel" name="calabel">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Testvalue 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caLabel" name="calabel_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Testvalue 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caLabel" name="calabel_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Testvalue 3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caLabel" name="calabel_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Testvalue 4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caLabel" name="calabel_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Testvalue 5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caLabel" name="calabel_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>450</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Testvalue 6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caLabel" name="calabel_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>cameter</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caLabel" name="calabel_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>catextentry_3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caLabel" name="calabel_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>canumeric</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caLabel" name="calabel_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>caspinbox</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caLabel" name="calabel_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>canlineedit</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caLabel" name="calabel_12">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>cathermo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="foreground">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </widget>
   <widget class="caToggleButton" name="catogglebutton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>92</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>caNumeric</class>
   <extends>QFrame</extends>
   <header>caNumeric</header>
  </customwidget>
  <customwidget>
   <class>caTextEntry</class>
   <extends>caLineEdit</extends>
   <header>caTextEntry</header>
  </customwidget>
  <customwidget>
   <class>caToggleButton</class>
   <extends>QCheckBox</extends>
   <header>caToggleButton</header>
  </customwidget>
  <customwidget>
   <class>caSpinbox</class>
   <extends>QFrame</extends>
   <header>caSpinbox</header>
  </customwidget>
  <customwidget>
   <class>caLabel</class>
   <extends>QLabel</extends>
   <header>caLabel</header>
  </customwidget>
  <customwidget>
   <class>caMeter</class>
   <extends>QWidget</extends>
   <header>caMeter</header>
  </customwidget>
  <customwidget>
   <class>caLineEdit</class>
   <extends>QLineEdit</extends>
   <header>caLineEdit</header>
  </customwidget>
  <customwidget>
   <class>caThermo</class>
   <extends>QWidget</extends>
   <header>caThermo</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
